cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -219,18 +219,18 @@ func startControllers(c *cloudcontrollerconfig.CompletedConfig, stop <-chan stru
 	go pvlController.Run(5, stop)
 	time.Sleep(wait.Jitter(c.ComponentConfig.Generic.ControllerStartInterval.Duration, ControllerStartJitter))
 
-	// Start the service controller
-	serviceController, err := servicecontroller.New(
+	// Start the loadbalancer controller
+	loadBalancerController, err := lbcontroller.New(
 		cloud,
-		client("service-controller"),
+		client("loadbalancer-controller"),
```

Comment:
this is externally visible as the name of the service account this controller runs as. is it worth the rename?

Reply:
+1

Reply:
It would seem confusing to have the controller named loadbalancer throughout the logs and code but called service on the "--controllers" flag on the command line. If you fix it in the controllers flag as well then there are more breaking changes.